### ISSUE TITLE
Support SNS PublishBatch

### ISSUE THREAD

**Issue by perrydunn** [Admiral-Piett/goaws#270]
The equivalent SQS SendMessageBatch is supported.

I'm happy to implement, please let me know?

**Comment by Admiral-Piett** [comment]
@perrydunn Good idea!  If you want to take on the implementation that would be awesome - feel free to assign this issue to yourself and make a PR?  If not, I'll get to it when I can.  :) 

For Reference: https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_SendMessageBatch.html

**Comment by perrydunn** [comment]
Sorry didn't mean to close! I don't appear to be able to assign this to myself but I am working on this now.

**Comment by Admiral-Piett** [comment]
@perrydunn Excellent!  Thank you!
